src/main/java/dao/FileReaderImpl.java

```diff
@@ -0,0 +1,26 @@
+package dao;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> readFile(String fileName) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader br = Files.newBufferedReader(Paths.get(fileName))) {
+            br.readLine();
```

Comment:
It seems that you are skipping the first line of the file unconditionally. If this is intended to skip a header, please ensure that all files processed by this method will have a header. Otherwise, you might be skipping important data. If the header is optional, you should add a condition to check whether the first line is a header or not.